samples/edgehog_app/README_NXP.md

```diff
@@ -0,0 +1,186 @@
+consequence you will need an Astarte instance to which the device will connect.
+The easiest way to set up one is to follow the
+[Astarte quick instance](https://docs.astarte-platform.org/device-sdks/common/astarte_quick_instance.html)
+tutorial.
+The tutorial guides you through setting up an Astarte instance on a local host machine. Both the
+host machine and the device board should be connected to the same LAN.
+
+Make sure you do not loose the following information:
+- Your Astarte realm name. The default for the Astarte quick instance is `test`.
+- The location of your Astarte instance. The default for the Astarte quick instance is an `astarte`
+  folder in your home.
+- The Astarte API endpoint. If you followed the Astarte quick instance guide you should have
+  choosen to configure Astarte for an external device. As a consequence this endpoint will be:
+  `api.astarte.<YOUR IP ADDRESS>.nip.io`.
+- A device ID registered using the dashboard and its associated credential secret. If you followed
+  the Astarte quick instance guide you can register a device in the dashboard, navigating to the
+  devices tab, clicking on register a new device and register a device with a random ID using the
+  default settings.
+
+### 3.2 Step 2
+
+In this step we will configure a local Edgehog instance on the same host machine that is running
+the Astarte instance.
+
+Start by cloning the Edgehog repository and navigate into it.
+```sh
+git clone https://github.com/edgehog-device-manager/edgehog -b v0.9.2 && cd edgehog
+```
+
+Next you will need to change some of the configuration variables in the `.env` file within the
+cloned repository. Replace the following variables with your own values.
+```sh
+DOCKER_COMPOSE_EDGEHOG_BASE_DOMAIN=edgehog.<YOUR IP ADDRESS>.nip.io
+DOCKER_COMPOSE_ASTARTE_BASE_DOMAIN=astarte.<YOUR IP ADDRESS>.nip.io
+SEEDS_REALM_PRIVATE_KEY_FILE=~/astarte/test_private.pem
+```
+The variables `DOCKER_COMPOSE_EDGEHOG_BASE_DOMAIN` and `DOCKER_COMPOSE_ASTARTE_BASE_DOMAIN` have
+been given values expecting your Astarte API endpoint to be `api.astarte.<YOUR IP ADDRESS>.nip.io`.
+If this was not the case and the Astarte API endpoint was for example `api.astarte.localhost` the
+two variables would be `edgehog.localhost` and `astarte.localhost` respectively.
+
+You can now bring up the docker containers required using docker compose.
+```sh
+docker compose up -d
+```
+
+Next, we will popuplate the Edgehog database.
+```sh
+docker compose exec edgehog-backend bin/edgehog eval Edgehog.Release.seed
+```
+Edgehog will automatically install its interfaces in our Astarte instance in a process called
+reconciliation.
+By design reconciliation happens each couple of minutes, to avoid waiting we can manually trigger
+a reconciliation using the command below.
+```sh
+docker compose exec edgehog-backend bin/edgehog rpc "Edgehog.Tenants.Tenant |> Ash.read! |> Enum.each(&Edgehog.Tenants.reconcile_tenant/1)"
+```
```

Comment:
Since [this pr](https://github.com/edgehog-device-manager/edgehog/pull/767/files) got merged do we still need this ?

Reply:
It's not yet included in any release. Since this readme uses `v0.9.2` of Edgehog it's still necessary.
We can remove it at the next patch of Edgehog